.github/workflows/push-to-npm-gpr.yml

```diff
@@ -28,3 +28,15 @@ jobs:
         run: npm publish --access public mitre-saf-v*.tgz
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      # Setup .npmrc file to publish to GitHub Package Registry
```

Comment:
where is the npmrc file?  what does it do?

Reply:
The file is created in the runner and includes the scope prefix, see these urls 

- https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
- https://docs.npmjs.com/cli/v9/configuring-npm/npmrc